Djikstra/Djikstra/Functions.cpp

```diff
@@ -0,0 +1,135 @@
+#include <vector>
+
+
+#include "Headers.h"
+
+
+namespace Graph
+{
+	bool Graph::node_exist( std::string name)
+	{
+		int i=0, vector_size;
+		bool exist = 0;
+		vector_size = m_vertex.size();
+		
+		while ( i < vector_size )
+		{
+			if (name ==  m_vertex[i].name ) //if name exist break loop, and function
+			{
+				exist = 1;
+				break;
+			}
+			i++;
+		}
+
+		return exist; //if vertex exist, then return 1;
+	}
+
+
+
+	void Graph::add_node( std::string name)
+	{
+		int vector_size;
+
+		vector_size = m_vertex.size();
+
+		m_vertex.resize(vector_size + 1 );
+		m_vertex[vector_size].name = name;
+	}
+
+
+
+	void Graph::add_edge( std::string name1, std::string name2, float dist )
+	{
+		int id1=NULL, id2=NULL, i;
```

Comment:
Avoid to use `NULL` - its value is never guaranteed in C++ (it may be `0`, but doesn't have to).